src/cli/utils/flags.go

```diff
@@ -58,6 +63,11 @@ const (
 	FileCreatedBeforeFN  = "file-created-before"
 	FileModifiedAfterFN  = "file-modified-after"
 	FileModifiedBeforeFN = "file-modified-before"
+
+	// Maintenance stuff.
+	MaintenanceSafetyFN = "safety"
+	QuickMaintenanceFN  = "quick"
```

Comment:
Since we're entertaining an enum instead of a flag for this elsewhere, I'd prefer `Mode` flag that defaults to `slow` (or whatever not-quick will be).  If we never add a third value, no big deal.  If we do, then it's not a breaking change to the cli.
```suggestion
	MaintenanceModeFN  = "mode"
```

Reply:
Some of these are Kopia terms and it may make sense to keep them.